docs/EDC-extension-configuration.md

```diff
@@ -0,0 +1,220 @@
+# Digital Twin Registry Data Plane HTTP Access Control Eclipse Dataspace Connector extension
+
+## 1 Introduction and goals
+
+The *Eclipse Dataspace Connector (EDC)* can provide access to many different backend services. These services have different security capabilities.
+
+### High level requirement
+The *Eclipse Dataspace Connector* extension can provide a granular access control solution for authorization of *Submodel* backend requests based on the registered
+*Submodel* endpoints of the *Digital Twins* which are available for the Data Consumer through the *Digital Twin Registry (DTR)*. 
```

Comment:
In AAS it is a Shell instead of Digital Twin.

Reply:
Thanks, I will do a find and replace!

Reply:
Done